src/foam/nanos/column_config/NestedPropertiesExpression.js

```diff
@@ -0,0 +1,217 @@
+/**
+ * @license
+ * Copyright 2020 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.column',
+  name: 'NestedPropertiesExpression',
+  extends: 'foam.mlang.AbstractExpr',
+
+  implements: ['foam.core.Serializable'],
+
+  documentation: 'Class for creating expression for a given nestedProperty ( e.g. address.countryId.name )',
+
+  javaImports: [
+    'foam.core.ClassInfo',
+    'foam.core.FObject',
+    'foam.core.PropertyInfo',
+    'foam.core.X',
+    'foam.mlang.Expr',
+    'foam.nanos.logger.Logger',
+    'java.lang.reflect.Method',
+    'static foam.mlang.MLang.*',
+  ],
+  properties: [
+    {
+      name: 'objClass',
+      class: 'Object',
+      javaType: 'foam.core.ClassInfo'
+    },
+    {
+      name: 'nestedProperty',
+      class: 'String'
+    }
+  ],
+  methods: [
+    {
+      name: 'f',
+      type: 'Any',
+      code: function(obj) {
+        var e =  this.returnDotExprForNestedProperty(this.objClass, this.nestedProperty.split('.'), 0);
+        if ( !e ) return null;
+        return e.f(obj);
+      },
+      javaCode: `
+        Expr e = returnDotExprForNestedProperty(getObjClass(), getNestedProperty().split("\\\\."), 0);
+        if ( e == null )
+          return null;
+        return e.f(obj);
```

Comment:
this expression context is different from next expression's context  ( which is the same as obj's context ). and this expression context allows to execute find methods but object context doesn't. and I'm not sure what's going on